feat(replay): Deprecate privacy API (#6174)
getsentry/sentry-docs: src/platforms/javascript/common/session-replay/configuration.mdx

## Start and Stop Recording

> Replay recording only starts when it is included in the `integrations` array when calling `Sentry.init` or calling `addIntegration` from the a Sentry client instance. To stop recording you can call the `stop()`:

Reviewer: ```suggestion
Replay recording starts if it's included in the `integrations` array when calling `Sentry.init` or, when `addIntegration` is called from a Sentry client instance. To stop recording, call `stop()`:
```
